tests/geo_commands_test.py

```diff
@@ -83,105 +87,143 @@ def test_geopos(redis):
 def test_georadius(redis):
     res = yield from redis.geoadd(
         'geodata',
-        13.361389, 38.115556,
-        'Palermo', 15.087269, 37.502669, 'Catania'
+        13.361389, 38.115556, 'Palermo', 15.087269, 37.502669, 'Catania'
     )
     assert res == 2
 
     res = yield from redis.georadius(
         'geodata', 15, 37, 200, 'km', encoding='utf-8'
     )
     assert res == [
-        ['Palermo'], ['Catania']
```

Comment:
i think command must return same result - list of namedtuples, because no reason for return list of strings, it's same record without additional data.